gitlab-ldap-sync.py

```diff
@@ -8,14 +8,23 @@
 import ldap.asyncsearch
 import logging
 
+CRON = False
+
+def logstdout(s):
+    if not CRON:
+        print(s)
+
 if __name__ == "__main__":
-    print('Initializing gitlab-ldap-sync.')
     config = None
```

Comment:
We should add a `global CRON` in both `logstdout` and `__main__` to ensure we use the same variable